core/src/program.rs

```diff
@@ -218,14 +223,12 @@ mod tests {
 
         let binary: Vec<u8> = parse_wat(wat);
 
-        let code = Code::try_new(
-            binary,
-            1,
-            None,
-            wasm_instrument::gas_metering::ConstantCostRules::default(),
-        )
+        let code = Code::try_new(binary, 1, |_| {
+            wasm_instrument::gas_metering::ConstantCostRules::default()
```

Comment:
I'd import `wasm_instrument::gas_metering::ConstantCostRules` according to idiomatin module import.
```suggestion
            ConstantCostRules::default()
```